AsyncDisplayKit/ASViewController.h

```diff
@@ -30,6 +30,9 @@ NS_ASSUME_NONNULL_BEGIN
 // Refer to examples/SynchronousConcurrency, AsyncViewController.m
 @property (nonatomic, assign) BOOL neverShowPlaceholders;
 
+/// Automatically adjust the range mode based on view evenets if the containing node confirms to the ASRangeControllerUpdateRangeProtocol
+@property (nonatomic, assign) BOOL automaticallyAdjustRangeModeBasedOnViewEvents;
```

Comment:
For now this should be a private or +Beta.h functionality as we test it, since ASDK public releases go out at a fairly regular cadence (I'm shipping 1.9.7 this weekend :) ).